source/frontend/src/features/properties/map/project/hooks/useAddProjectFormManagement.tsx

```diff
@@ -0,0 +1,41 @@
+import { FormikHelpers } from 'formik';
+import { useProjectProvider } from 'hooks/providers/useProjectProvider';
+import { Api_Project } from 'models/api/Project';
+import { useCallback } from 'react';
+
+import { AddProjectYupSchema } from '../add/AddProjectFileYupSchema';
+import { ProjectForm } from '../add/models';
+
+export interface IUseAddProjectFormManagementProps {
+  /** Optional - callback to execute after acquisition file has been added to the datastore */
+  onSuccess?: (project: Api_Project) => void;
+  initialForm?: ProjectForm;
+}
+
+export function useAddProjectFormManagement(props: IUseAddProjectFormManagementProps) {
```

Comment:
nit: useAddProjectForm reads just the same

Reply:
please expand.

Reply:
Just the name of the component. Given that is a form that is a dding a project addProject vs management sounds better